public/locales/tr/send.ftl

```diff
@@ -15,6 +16,7 @@ encryptingFile = Şifreleniyor...
 decryptingFile = Şifre çözülüyor...
```

Comment:
That's on locales to decide. Having said that, I fixed Italian because I was fooled by translation memory matches, and didn't realize it was 3 separate dots in Pontoon's interface.